.github/workflows/tests.yml

```
@@ -36,8 +35,8 @@ jobs:
         # Workaround to map x.y Python version to form test-xy
         # https://stackoverflow.com/a/67248310
         run: |
-          RAW_TOX_ENV="test-py${{ matrix.python-version }}"
-          TOX_ENV=$(echo $RAW_TOX_ENV | sed 's/\.//')
+          RAW_TOX_ENV="test-py${{ matrix.python-version }}-dolfinx${{ matrix.dolfinx-version }}"
+          TOX_ENV=$(echo $RAW_TOX_ENV | sed 's/\.//' | sed 's/\.//' )
```

Have to do this twice now because the `RAW_TOX_ENV` is something like `test-py3.9-dolfinx0.6` (two `\.`'s to be replaced).

---

```suggestion
          TOX_ENV=$(echo $RAW_TOX_ENV | sed 's/\.//g')
```
Should I think have the same effect without the duplication

---

Yep! Good spot. Still running ×4 in the matrix. 👍